www/front_src/src/Authentication/Form/PasswordBlockingPolicy/index.test.tsx

```diff
@@ -0,0 +1,238 @@
+} from '../defaults';
+import {
+  labelBlockingDurationMustBeLessThanOrEqualTo7Days,
+  labelBlockingTimeBeforeNewConnectionAttempt,
+  labelChooseAValueBetween1and10,
+  labelDay,
+  labelGood,
+  labelMinutes,
+  labelNumberOfAttemptsBeforeBlockingNewAttempts,
+  labelPasswordBlockingPolicy,
+  labelStrong,
+  labelThisWillNotBeUsedBecauseNumberOfAttemptsIsNotDefined,
+  labelWeak,
+} from '../../translatedLabels';
+
+import PasswordCasePolicy from '.';
+
+const noOp = jest.fn();
+
+interface Props {
+  initialValues: SecurityPolicy;
+}
+
+const TestComponent = ({ initialValues }: Props): JSX.Element => {
+  const validationSchema = useValidationSchema();
+
+  return (
+    <Formik<SecurityPolicy>
+      enableReinitialize
+      validateOnBlur
+      validateOnMount
+      initialValues={initialValues}
+      validationSchema={validationSchema}
+      onSubmit={noOp}
+    >
+      {(): JSX.Element => <PasswordCasePolicy />}
+    </Formik>
+  );
+};
+
+const renderPasswordBlockingPolicy = (
+  initialValues: SecurityPolicyFromAPI = defaultSecurityPolicy,
+): RenderResult =>
+  render(
+    <TestComponent initialValues={initialValues.password_security_policy} />,
+  );
+
+describe('Password Blocking Policy', () => {
+  it('renders the password blocking policy fields with values', async () => {
+    renderPasswordBlockingPolicy();
+
+    await waitFor(() => {
+      expect(screen.getByText(labelPasswordBlockingPolicy)).toBeInTheDocument();
+    });
+
+    expect(
+      screen.getByLabelText(labelNumberOfAttemptsBeforeBlockingNewAttempts),
+    ).toHaveValue(5);
+
+    expect(
+      screen.getByText(labelBlockingTimeBeforeNewConnectionAttempt),
+    ).toBeInTheDocument();
+
+    expect(
+      screen.getByLabelText(
+        `${labelBlockingTimeBeforeNewConnectionAttempt} ${labelMinutes}`,
+      ),
+    ).toHaveTextContent('15');
+
+    expect(screen.getByText(labelWeak)).toBeInTheDocument();
+  });
+
+  it('displays an error message when the number of attempts is 0', async () => {
+    renderPasswordBlockingPolicy();
+
+    await waitFor(() => {
+      expect(screen.getByText(labelPasswordBlockingPolicy)).toBeInTheDocument();
+    });
+
+    userEvent.type(
+      screen.getByLabelText(labelNumberOfAttemptsBeforeBlockingNewAttempts),
+      '0',
+    );
+
+    await waitFor(() => {
+      expect(
+        screen.getByText(labelChooseAValueBetween1and10),
+      ).toBeInTheDocument();
+    });
+  });
+
+  it('displays an error message when the number of attempts is 11', async () => {
+    renderPasswordBlockingPolicy();
+
+    await waitFor(() => {
+      expect(screen.getByText(labelPasswordBlockingPolicy)).toBeInTheDocument();
+    });
+
+    userEvent.type(
+      screen.getByLabelText(labelNumberOfAttemptsBeforeBlockingNewAttempts),
+      '11',
+    );
+
+    await waitFor(() => {
+      expect(
+        screen.getByText(labelChooseAValueBetween1and10),
+      ).toBeInTheDocument();
+    });
+  });
```

Comment:
I would merge these two cases into one to optimize the tests runtime. We can use a name like: "it displays an error message when the number of attemps is outside the bounds"